SciPy2024/module2/3-retrieval-augmented-text-generation.ipynb

```diff
@@ -9,79 +9,22 @@
    ]
```

Comment:
<p>Include link to "evaluation that we did" <a href="https://github.com/uw-ssec/tutorials/issues/6" target="_blank">chore: Research on the best embedding models for the tutorial · Issue #6 · uw-ssec/tutorials (github.com)</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>